fleetctl/fleetctl.go

```diff
@@ -669,6 +669,81 @@ func createUnit(name string, uf *unit.UnitFile) (*schema.Unit, error) {
 	return &u, nil
 }
 
+// checkReplaceUnitState checks if the unit should be replaced
+// It takes a Unit object as a parameter
+// It returns 0 on success and if the unit should be replaced, 1 if the
+// unit should not be replaced; and any error acountered
+func checkReplaceUnitState(unit *schema.Unit) (int, error) {
```

Comment:
> Please follow proper orthography: sentences always begin with a capital letter, and end in a full stop.

Basically I agree with the principle. But in this case I'd rather not capitalize the beginning letter, because of confusion during pattern matching in the editor. I'd rather rewrite the sentence, not to start a sentence with a function name.

Besides I would normally just leave comments written by others, given the fact that every comment-writing already requires a fair amount of effort. Think about other bunch of functions without any comment. A wrong-spelled comment is much more valuable than no comment. ;-)